### PR TITLE
docs: add explanation about publishing a slice

### PR DESCRIPTION
**Pull request by SamirPS**
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?

-----

### CODE REVIEW

File: docs/how-to/index.rst [hunk 1]
```diff
@@ -13,3 +13,4 @@ They'll help you achieve an end result but may require you to understand and ada
 
    Get started - quick guide <get-started>
    Build the documentation <build-docs>
+   Publish your slice for everyone <publish-slice>
```
Comment (cjdcordeiro) [inline docs/how-to/index.rst:16]:
```suggestion
   Release a slice definitions file <publish-slice>
```

File: docs/how-to/publish-slice.rst [hunk 2]
```diff
@@ -0,0 +1,10 @@
+How to make my slice definitions available for everyone?
```
Comment (cjdcordeiro) [inline docs/how-to/publish-slice.rst:1]:
```suggestion
How to make custom slice definitions available for everyone?
```

File: docs/how-to/publish-slice.rst [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+********************************************************
+
+So now, you have created and tested your slice and want to make it **available for everyone.**
```
Comment (cjdcordeiro) [inline docs/how-to/publish-slice.rst:4]: Could you please elaborate a bit more? This does tell people what to do, but with very little context.

**Review by cjdcordeiro — CHANGES_REQUESTED** [review]

File: docs/how-to/publish-slice.rst (hunk 3 above)
Comment (SamirPS) [reply docs/how-to/publish-slice.rst]: Done

File: docs/how-to/publish-slice.rst [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+So right now, we have create a custom package slice and we have install it into a rocks.
+Nevertheless, you saw in the previous tutorial that you need to call chisel into your rocks because your slice is not available for everyone.
+In this tutorial we will see how to make it **accessible to everyone.**
+
+#. Create a fork of this repository: https://github.com/canonical/chisel-releases
+#. Create a branch and add your modification
+#. Create a pull request
+#. Wait for PR to be merged
+
+And that's it! You're custom slice definitions is available for everyone **Congrats!**
```
Comment (cjdcordeiro) [inline docs/how-to/publish-slice.rst:13]:
```suggestion
And that's it! Your custom Chisel release and new slice definitions are now available in Chisel, and anyone can use them. **Congrats**! And thank you for your contribution.
```

**Review by cjdcordeiro — CHANGES_REQUESTED** [review]

File: docs/how-to/publish-slice.rst [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+At this stage, you have created some package slice definitions and you have a custom Chisel release in your local development environment.
+You have also tested this custom Chisel release, and it works! You believe there are others who could really use it as well, so **how can you make it accessible to everyone**?
+
+It is as simple as proposing your changes into the upstream `Chisel releases repository <https://github.com/canonical/chisel-releases>`_:
+
+Create a fork of this repository: https://github.com/canonical/chisel-releases::
+
+    # Let's assume we are working with Ubuntu 22.04
+    git clone -b ubuntu-22.04 https://github.com/canonical/chisel-releases/
```
Comment (cjdcordeiro) [inline docs/how-to/publish-slice.rst:12]: this is not a fork

File: docs/how-to/publish-slice.rst [hunk 6]
```diff
@@ -0,0 +1,23 @@
+How to make custom slice definitions available for everyone?
+************************************************************
+
+At this stage, you have created some package slice definitions and you have a custom Chisel release in your local development environment.
+You have also tested this custom Chisel release, and it works! You believe there are others who could really use it as well, so **how can you make it accessible to everyone**?
+
+It is as simple as proposing your changes into the upstream `Chisel releases repository <https://github.com/canonical/chisel-releases>`_:
+
+Create a fork of this repository: https://github.com/canonical/chisel-releases::
```
Comment (cjdcordeiro) [inline docs/how-to/publish-slice.rst:9]: can you please bring the bullet list from before? you have nested ":" in this line now

File: docs/how-to/publish-slice.rst [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+    git clone -b ubuntu-22.04 https://github.com/canonical/chisel-releases/
+
+Create a branch and add your modification::
+
+    git checkout -b myopensslslice
+    # make changes
+    git commit
+    git push
+
+After that create a pull request and wait for your pull request to be merged
```
Comment (cjdcordeiro) [inline docs/how-to/publish-slice.rst:21]:
```suggestion
After that, create a pull request and wait for it to be merged.
```

File: docs/how-to/publish-slice.rst [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+Create a fork of this repository: https://github.com/canonical/chisel-releases::
+
+    # Let's assume we are working with Ubuntu 22.04
+    git clone -b ubuntu-22.04 https://github.com/canonical/chisel-releases/
+
+Create a branch and add your modification::
+
+    git checkout -b myopensslslice
+    # make changes
+    git commit
```
[the remark this answers is not in the document]
Comment (SamirPS) [reply docs/how-to/publish-slice.rst:18]: how it's doesn't work i don't understand

File: docs/how-to/publish-slice.rst [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+    git push
```
[the remark this answers is not in the document]
Comment (SamirPS) [reply docs/how-to/publish-slice.rst:19]: how it's doesn't work i don't understand

File: docs/how-to/publish-slice.rst [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+It is as simple as proposing your changes into the upstream `Chisel releases repository <https://github.com/canonical/chisel-releases>`_:
+
+#. Create a fork of https://github.com/canonical/chisel-releases and clone this repository:
+
+    .. code-block:: bash
+
+        # Let's assume we are working with Ubuntu 22.04
+        $ git clone -b ubuntu-22.04 https://github.com/<your_github_username>/chisel-releases/
+
+#. Create a branch and add your modification:
```
Comment (cjdcordeiro) [inline docs/how-to/publish-slice.rst:16]:
```suggestion
#. Add your modifications:

    .. code-block:: bash
        
        $ cp <path/to/your/slice/definitions/file> chisel-releases/slices

#. Create a new branch, commit and push it to your fork:
```

File: docs/how-to/publish-slice.rst [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+
+    .. code-block:: bash
+
+        $ git checkout -b myopensslslice
```
Comment (cjdcordeiro) [inline docs/how-to/publish-slice.rst:20]:
```suggestion
        $ cd chisel-releases
        $ git checkout -b create-openssl-bins-slice
```

File: docs/how-to/publish-slice.rst [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+        # make changes
+        $ git add .
+        $ git commit -m "Add slice definition file for 'name_of_the_package'"
```
Comment (cjdcordeiro) [inline docs/how-to/publish-slice.rst:23]:
```suggestion
        $ git commit -m "feat: add new slice definitions for 'name_of_the_package'"
```

**Review by cjdcordeiro — CHANGES_REQUESTED** [review]

File: docs/how-to/publish-slice.rst [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+        $ git clone -b ubuntu-22.04 https://github.com/<your_github_username>/chisel-releases.git
+
+#. Create a branch:
+
+    .. code-block:: bash
+
+        $ cd chisel-releases
+        $ git checkout -b create-openssl-bins-slice
+
+#. Add your modification:
```
Comment (cjdcordeiro) [inline docs/how-to/publish-slice.rst:23]:
```suggestion
#. Add and commit your modifications:
```

File: docs/how-to/publish-slice.rst [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+
+    .. code-block:: bash
+
+        $ cp <path/to/your/slice/definitions/file> chisel-releases/slices
```
Comment (cjdcordeiro) [inline docs/how-to/publish-slice.rst:27]: this will fail with the wrong path

**Review by cjdcordeiro — CHANGES_REQUESTED** [review]

**Comment by SamirPS** [comment]
The step to publish a slice have been tested and approved by me

**Review by cjdcordeiro — APPROVED** [review]
lgtm

**Review by cjdcordeiro — APPROVED** [review]

File: docs/how-to/publish-slice.rst [hunk 15]
```diff
@@ -0,0 +1,36 @@
+.. _publishslice:
+
+How to make custom slice definitions available for everyone?
```
Comment (tigarmo) [inline docs/how-to/publish-slice.rst:3]:
```suggestion
How to make custom slice definitions available for everyone
```

Same as the install-slice how-to; to keep consistent with the create-slice one

File: docs/how-to/publish-slice.rst [hunk 16]
```diff
@@ -0,0 +1,36 @@
...
+************************************************************
+
+At this stage, you have created some package slice definitions and you have a custom Chisel release in your local development environment.
+You have also tested this custom Chisel release, and it works! You believe there are others who could really use it as well, so **how can you make it accessible to everyone**?
+
+It is as simple as proposing your changes into the upstream `Chisel releases repository <https://github.com/canonical/chisel-releases>`_:
+
+#. Fork this repository https://github.com/canonical/chisel-releases and clone your repository:
```
Comment (tigarmo) [inline docs/how-to/publish-slice.rst:11]:
```suggestion
#. Fork this repository https://github.com/canonical/chisel-releases and clone your fork:
```

**Review by tigarmo — CHANGES_REQUESTED** [review]

**Review by tigarmo — APPROVED** [review]
thanks!

**Pull request merged**